extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -17,7 +28,25 @@ define('ext.wikia.adEngine.template.porvata', [
 	 * @param {string} [params.vastUrl] - Vast URL (DFP URL with page level targeting will be used if not passed)
 	 */
 	function show(params) {
+		if (params.vpaidMode === 2) {
+			params.container = getVideoContainer(params.slotName);
+		}
+
 		porvata.inject(videoSettings.create(params)).then(function (video) {
+			if (params.vpaidMode === 2) {
+				video.addEventListener('loaded', function () {
+					var ad = video.ima.getAdsManager().getCurrentAd(),
+						contentType = ad.getContentType() || '';
+
+					if (contentType.indexOf('application/') !== -1) {
```

Comment:
what about moving `application/` to const as well with some self explanatory name or create a function to check this `isApplicationType()` but this can be an overhead though